.codecov.yml

```diff
@@ -1,5 +1,8 @@
 coverage:
   status:
+    patch:
```

Comment:
open the informational mode for new code checking, https://docs.codecov.com/docs/commit-status#informational

cc @qiujian16 @xuezhaojun @clyang82